src/test/test-varlink-idl.c

```diff
@@ -24,9 +24,9 @@
 #include "varlink-io.systemd.PCRLock.h"
 #include "varlink-io.systemd.Resolve.h"
 #include "varlink-io.systemd.Resolve.Monitor.h"
+#include "varlink-io.systemd.Service.h"
 #include "varlink-io.systemd.UserDatabase.h"
 #include "varlink-io.systemd.oom.h"
-#include "varlink-io.systemd.service.h"
```

Comment:
hmm, changing this to lowercase might be a bit too late, we already released it like this in a couple of versions.

I used the low-level spelling for this because i wanted to mimic Varlink's generic introspection interface, i.e.:

https://varlink.org/Service

that also uses the lowercase spelling.

yeah, this is a bit unfortunate, but i'd not needlessly break interface here